cumulusci/core/config/org_config.py

```diff
@@ -254,7 +263,7 @@ def populate_expiration_date(self):
     @property
     def organization_sobject(self):
         """Cached copy of Organization sObject. Does not perform API call."""
-        return self._org_sobject
+        return getattr(self, "_org_sobject", None)
```

Comment:
This code probably relied previously on implicit None for config objects which is evil.